device/celestica/x86_64-cel_e1031-r0/sonic_platform/common.py

```diff
@@ -184,12 +183,13 @@ def write_txt_file(self, file_path, value):
             return False
         return True
 
-    def is_host(self):
-        try:
-            subprocess.call(self.HOST_CHK_CMD, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
-        except FileNotFoundError:
-            return False
-        return True
+    def is_host():
+        """
+        Test whether current process is running on the host or an docker
+        return True for host and False for docker
+        """
+        docker_env_file = '/.dockerenv'
+        return os.path.exists(docker_env_file) is False

```

Comment:
Minor change: `os.path.exists(docker_env_file) is False` -> `not os.path.exists(docker_env_file)`